### ISSUE TITLE
Markdown preview inconsistent with render when using _Italics_ 

### ISSUE THREAD

**Issue by amirgon** [ASKBOT/askbot-devel#573]
This issue appears frequently on our local AskBot site.
Many posts are written by C programmers who write C macro names as part of their questions/answers. Macros are frequently built from words separated by underscore.

When writing the post, on the MarkDown preview **underscores remain and does not convert to italics**. However, after posting we discover that underscore were removed from macro names and parts of the names were converted to italics.

### The main issue here is the inconsistency between the preview of the post and the final result.  

Examples:

---

When posting:

    TEST1_TEST2_TEST3  

**Preview** shows:

> TEST1_TEST2_TEST3  

But after posting, the post looks like this:

> TEST1*TEST2*TEST3 

(TEST2 is displayed in italics)

----

When posting
```
TEST1_TEST2  
TEST1_TEST2  
TEST1_TEST2
```

**preview shows it correctly** (without any italics). But after posting I get:
![askbot_test1_test2](https://cloud.githubusercontent.com/assets/11742638/14638795/bcee814e-0641-11e6-8b80-a60aa2e068d9.png)

**Comment by evgenyfadeev** [comment]
True, preview and server side markdown use different algorithms, however this specific issue is resolved by enabling "code-friendly Markdown" in the "settings" -> "markup in posts" using administrator user profile.
